src/__tests__/services/UserProfileServiceTest.js

```diff
@@ -0,0 +1,58 @@
+import axios from "axios";
+import {UserProfile} from "../../services/UserProfileService";
+
+jest.mock("axios");
+
+describe("UserProfileService", () => {
+    describe("getByAlias()", () => {
+        it("makes a GET request to the backend API", async () => {
+            axios.get.mockResolvedValue({
+                data: {
+                    response: [{
+                        role: {}
+                    }]
+                }
+            });
+
+            await UserProfile.getByAlias("ExampleAlias");
+
+            expect(axios.get).toHaveBeenCalledTimes(1);
+            expect(axios.get).toHaveBeenCalledWith("http://localhost:8080/api/user/v1/profiles/?alias=ExampleAlias");
```

Comment:
Probably best to use the config value here. Don't want the tests to fail if for some reason the API URL changes.

Reply:
Good point!